### ISSUE TITLE
Response based on content of JWT (parsing JWTs)

### ISSUE THREAD

**Issue by TobiasJu** [mockoon/mockoon#774]
**Target**:
My goal is to mock an authentification request with Mockoon, and give the user a special role, depending on his username.
  
**Background**:
In our production system we have an oauth instance, which we do not have in our dev env (here we only have a keycloak). And we do not want to add more REST requests.
  
**Feature request**:
Our client sends an JWT (in the body of a POST request) to Mockoon, which then sends the user role back to the client. Now it would be nice, if Mockoon could:

- parse the base64 encoded JWT 
- extract the userName property 
- changes the response based on the userName

eg:
```  
{
  "userRole": 
    {{#switch (body.parseJWT 'userName')}}
      {{#case 'John'}}"EmergencyDoc"{{/case}}
      {{#case 'Jack'}}"Admin"{{/case}}
      {{#default}}"defaultClient"{{/default}}
    {{/switch}}
}
```

**Comment by stefan-prange** [comment]
I'd love to see such feature, but I'd prefer a solution which was independent from the body object.
You should be able to pass JWT token's from an arbitrary origin to that function.
Note that JWT tokens are usually transferred in a request header like "Authorization: Bearer <serialized JWT token>".

So a helper function called for instance "jwtPayload" would be great. It could take 2 parameters:
Parameter 1: the serialized JWT token
Parameter 2: The name of the payload attribute to extract.

Example: 
The helper function call
```
jwtPayload "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c" "sub"
```
would result in the value `1234567890`. 

Besides a function `jwtPayload` a function `jwtHeader` would also be useful for some people, although most use cases deal with the token's payload, not with its header attributes.

**Comment by 255kb** [comment]
📦 This was released in [v8.4.0](https://mockoon.com/releases/8.4.0/)
